scripts/pi-hole/js/db_lists.js

```diff
@@ -84,12 +84,10 @@ function updateTopClientsChart() {
                 {
                     var idx = client.indexOf("|");
                     clientname = client.substr(0, idx);
-                    clientip = client.substr(idx+1, client.length-idx);
```

Comment:
This code is likely a copy-paste left over from the live data where both, IP and name are available. In ling-term data, only the IP is available.

@XhmikosR We could go further and remove `var url = clientname;` below and directly use `clientname` (instead of `url`) in https://github.com/pi-hole/AdminLTE/pull/1093/files#diff-550ed2961432528888f025e53a533bcdL97

Reply:
@DL6ER feel free to push to this branch or the xo branch (I'd prefer this and then we cherry pick it) so that we can test things